src/components/common/passcode/passcode.modal.create.js

```diff
@@ -34,16 +37,26 @@ class CreatePasscodeModal extends PureComponent {
     this.baseModal.resetModal(flow.title);
   }
 
+  handleNewPasscode = (newPasscode) => {
```

Comment:
why repeated code?

Reply:
Do you mean because of the code that is in the reset screen?
It's not good I agree, but even the indexes are different between screens so trying to re-use the code here would end up being more confusing IMO.
To reuse it I think we should make a bigger refactor of this section

Reply:
Ok, got it. This are the things we should consider in a near future.